platform/xbox/functions/_PDCLIB/_PDCLIB_w32errno.c

```diff
@@ -0,0 +1,54 @@
+/* _PDCLIB_w32errno( void )
+
+   This file is part of the Public Domain C Library (PDCLib).
+   Permission is granted to use, modify, and / or redistribute at will.
+*/
+
+#ifndef REGTEST
+#include <errno.h>
+#include <wchar.h> // Watcom bug: winnt.h assumes string.h defines wchar_t

```

Comment:
What does this mean? Should it be important to us?

Reply:
No, that's only relevant for Watcom compiler users. That comment is only there because I copied the file and left it untouched.